### PR TITLE
Wallet/Account pair

### PR DESCRIPTION
**Pull request by jmatosp**
This will allow to use a valida Wallet/Account pair in validator, reducing the dependency on AccountManager()

No impact on existing functionality

### CODE REVIEW

File: accounts/wallet_account.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	Account() Account
+}
+
+type walletAccount struct {
+	Wallet
+	account Account
+}
+
+var (
+	errInvalidAccountAddress = errors.New("invalid account address, doesnt exists in wallet")
```
Comment (rgeraldes) [inline accounts/wallet_account.go]: we should move this one to the beginning of the file - they follow that convention in go-eth afaik

Comment (rgeraldes) [reply accounts/wallet_account.go]: also I think that it would be important to show the the account address that is invalid.

Comment (jmatosp) [reply accounts/wallet_account.go]: :+1: 
will make the changes

File: accounts/wallet_account.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+func (err errInvalidAccountAddress) Error() string {
+	return fmt.Sprintf("invalid account address, doesnt exists in wallet: %s" + err.account.Address.String())
+}
+
+func newErrInvalidAccountAddress(account Account) errInvalidAccountAddress {
+	return errInvalidAccountAddress{account}
+}
+
```
Comment (rgeraldes) [inline accounts/wallet_account.go]: noticed just know that the accounts package contains an errors.go file with all the errors.

Comment (jmatosp) [reply accounts/wallet_account.go]: will move there

**Pull request merged**